README.md

```diff
@@ -99,12 +99,16 @@ virtual machine or install several dependencies.
 
 1. Install Vagrant and VirtualBox on your system.
 2. Clone the repository
-3. Navigate to the cloned directory :
+   
+   ```
+   https://github.com/p4lang/tutorials.git
+   ```
+4. Navigate to the cloned directory :
```

Comment:
Why are you omitting step number 3?

Reply:
To clarify, the changes as I am reviewing them now change a list of steps numbered 1,2,3,4 to a list of steps numbered 1,2,4,5, which does not look correct to me.